azure-pipelines.release.yml

```diff
@@ -44,28 +44,6 @@ steps:
       yarn generate-version-files
     displayName: Generate version files
 
-  # The contents of this file is configured in the component governence tab of ADO
```

Comment:
OSSCELA has confirmed we don't need to actually include the NOTICE file in packages, running the enhanced license scans should be enough